zmsadmin/js/page/availabilityDay/index.js

```
@@ -195,100 +138,74 @@ class AvailabilityPage extends Component {
-                        ...cleanupAvailabilityForSave(sendAvailability),
-                        kind: availability.kind || 'default'
-                    }
-                ],
-                selectedDate: selectedDate
-            };
+            let list = [availability];
+            const sendData = list.map(availability => {
+                const sendAvailability = Object.assign({}, availability)
+                if (availability.tempId) {
+                    delete sendAvailability.tempId
+                }
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)